src/index.html

```diff
@@ -11,12 +11,60 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="/"
+        class="logo"
+      >
+        <img
+          src="images/logo.png"
+          alt="Logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="#apple"
+              class="nav__link is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li><a href="https://example.com/moyo.samsung.html">Samsung</a></li>
```

Comment:
Fix code style everywhere and add classes for tags `li` and `a`
```suggestion
          <li class="nav__item">
            <a class="nav__link" href="https://example.com/moyo.samsung.html">Samsung</a>
          </li>
```

Reply:
Also, need to fix the href attribute as in the first link